enterprise/server/scheduling/scheduler_server/scheduler_server.go

```diff
@@ -1144,10 +1144,13 @@ func (s *SchedulerServer) claimTask(ctx context.Context, taskID string, claimTim
 		// Success
```

Comment:
do we also want to log on line 1134?

```go
	r, err := redisAcquireClaim.Run(ctx, s.rdb, []string{s.redisKeyForTask(taskID)}).Result()
	if err != nil {
		log.CtxErrorf(ctx, "claimTask error: redis script failed: %s", err)
		return err
	}
```

maybe 1139 too, logging the type info

```go
	c, ok := r.(int64)
	if !ok {
		log.CtxErrorf(ctx, "unexpected result from claim attempt: %v (type %T)", r, r)
		return status.FailedPreconditionErrorf("unexpected result from claim attempt: %v (type %T)", r, r)
	}
```